packages/prisma/zod-utils.ts

```diff
@@ -123,9 +123,7 @@ export const bookingResponses = z
       nonEmptyString(),
       z.object({
         firstName: nonEmptyString(),
-        lastName: nonEmptyString()
-          .refine((value: string) => value.trim().length > 0)
-          .optional(),
+        lastName: z.string().optional(),
```

Comment:
`lastName` was not configured properly. It was marked optional as well as it was required to have 1 char at least.